generators/cmd/cli/cmd/write.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"generators/pkg/writer"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var specFilename string
+var outputFilename string
+var writeCmd = &cobra.Command{
+	Use:   "write",
+	Short: "Write generated configuration to disk.",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(specFilename) == 0 {
+			cmd.Help()
+			return nil
```

Comment:
We probably want to return an error, since it will cause the program to return a non-zero exit code, which will signal to the user that there was an issue

IIRC, cobra will automatically print the help func if an error is returned, so we should be able to do something like:

```go
if len(specFilename) == 0 {
  return errors.New("no input file specified")
}
```